.github/dependabot.yml

```
@@ -0,0 +1,16 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
```

This is what you use for poetry---see https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem

---

good point, will add a comment

```suggestion
  - # "pip" is the correct setting for poetry, per https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
    package-ecosystem  package-ecosystem: "pip"
```

---

Ahh good idea. (Was just double checking. I vaguely remember reading those docs before!)